extensions/observability-devservices/deployment/src/main/java/io/quarkus/observability/deployment/DevResourcesBuildItem.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.observability.deployment;
+
+import io.quarkus.builder.item.SimpleBuildItem;
+
+final class DevResourcesBuildItem extends SimpleBuildItem {
```

Comment:
Afair this is a sort of a dummy build item, since BuildStep requires an item to execute process  -- either via builder or returning build item.

Using service locator, we already return DevServicesResultBuildItem, actually a dev service wrapped in it.